app/models/concerns/act_as_taggable_on.rb

```diff
@@ -0,0 +1,28 @@
+module ActAsTaggableOn
+  def acts_as_taggable_on
+    mapping_table_name = "#{name.underscore}_tags".to_sym
```

Comment:
Could we just use `def self.tagging_relation_name` for this?  Any reason for this to be a local variable vs a method?

Reply:
hm I think I can't use the method for the has_many? And I can use the variable in the methods body... Or can I?

Reply:
You can you just need to define the method in the `class_eval` before you do the `has_many`.  Would prefer that over having it defined in two places two different ways.

Reply:
done